src/plugins/data/public/query/query_string/dataset_service/types.ts

```diff
@@ -74,7 +94,7 @@ export interface DatasetTypeConfig {
    * Retrieves the search options to be used for running the query on the data connection associated
    * with this Dataset
    */
-  getSearchOptions?: () => DatasetSearchOptions;
+  getSearchOptions?: (dataset: Dataset) => DatasetSearchOptions;
```

Comment:
Do we need this change? I dont see it being used anywhere

Reply:
Will remove